.github/workflows/deploy_dev.yml

```diff
@@ -6,9 +6,9 @@ name: 'Dev Deploy'
 
 on:
   pull_request_review:
-    types: [submitted]
+    types: [submitted, edited] # submitted, reviewer edits the PR (ex comments)
   workflow_dispatch: # This enables manual triggers
```

Comment:
I think workflow_dispatch will work here since the job should be allowed to continue with 
 ` if: github.event.review.state == 'approved' && github.event.pull_request.base.ref == 'main'`